repos/fdbt-site/cypress_tests/cypress/support/helpers.ts

```diff
@@ -29,10 +29,12 @@ export const clickElementByText = (text: string): Cypress.Chainable<JQuery> => g
 export const getRandomNumber = (min: number, max: number): number => Cypress._.random(min, max);
 
 export const getHomePage = (noc = 'LNUD'): void => {
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    cy.clearCookies({ domain: null });
     cy.visit(`?disableAuth=${noc}`);
+    cy.url().then((url) => {
+        if (url.includes('?disableAuth')) {
+            clickElementById('start-now-button');
```

Comment:
When will it not include '?disableAuth' ? Doesnt the line above always send the user there.